### ISSUE TITLE
Test failure with mergeOverlappingRanges

### ISSUE THREAD

**Issue by C10ne** [spatie/opening-hours#163]
Hi,
After the update to v2.10.0 mergeOverlappingRanges stopped supporting hours array with multiple elements.
I've verified this behaviour with freshly pulled repository and modifying the test:

```
public function testMergeOverlappingSupportsHoursAndIgnoreData()
    {
        $data = OpeningHours::mergeOverlappingRanges([
             'monday' => [
                 'hours' => [
                     '09:00-12:00',
                     '13:00-18:00',
                 ],
             ],
         ]);

        $monday = OpeningHours::create($data)->forDay('Monday');

        $this->assertNull($monday->getData());
        $this->assertNull($monday[0]->getData());
        $this->assertSame('09:00-23:00', (string) $monday);
    }
```
Running the test results in:

```1) Spatie\OpeningHours\Test\OpeningHoursTest::testMergeOverlappingSupportsHoursAndIgnoreData
Spatie\OpeningHours\Exceptions\InvalidDayName: Day `0` isn't a valid day name. Valid day names are lowercase english words, e.g. `monday`, `thursday`.

/home/vagrant/projects/opening-hours/src/Exceptions/InvalidDayName.php:9
/home/vagrant/projects/opening-hours/src/OpeningHours.php:639
/home/vagrant/projects/opening-hours/src/OpeningHours.php:589
/home/vagrant/projects/opening-hours/src/OpeningHours.php:178
/home/vagrant/projects/opening-hours/src/OpeningHours.php:66
/home/vagrant/projects/opening-hours/tests/OpeningHoursTest.php:1420

**Comment by kylekatarnls** [comment]
Thanks for the report. I confirm it's a regression. Should be fixed in 2.10.1

**Comment by kylekatarnls** [comment]
[**2.10.1** released](https://github.com/spatie/opening-hours/releases/tag/2.10.1).
